WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -614,7 +614,8 @@ private extension OrderDetailsDataSource {
 
     private func configureCollectPaymentButton(cell: ButtonTableViewCell, at indexPath: IndexPath) {
         cell.configure(style: .primary,
-                       title: Titles.collectPayment) { [weak self] in
+                       title: Titles.collectPayment,
+                       accessibilityIdentifier: "order-detail-collect-payment-button") { [weak self] in
```

Comment:
super nit: might be nice to be consistent with other accessibility identifiers in order details
```suggestion
                       accessibilityIdentifier: "order-details-collect-payment-button") { [weak self] in
```